corehq/apps/app_manager/views.py

```diff
@@ -916,6 +917,25 @@ def get_details():
         }
 
 
+def _get_default_media_file_name(app, module, form):
+
+    default_file_name = ""
+
+    if module:
+        if trans(module.name, app.langs) == _("Untitled Module"):
```

Comment:
since this is now being used in logic it might make sense to pull it out into a constant